apps/ledger-live-desktop/src/renderer/components/ExportLogsButton.jsx

```diff
@@ -15,7 +15,11 @@ import type { Props as ButtonProps } from "~/renderer/components/Button";
 import { accountsSelector } from "~/renderer/reducers/accounts";
 
 const saveLogs = async (path: { canceled: boolean, filePath: string }) => {
-  await ipcRenderer.invoke("save-logs", path, JSON.stringify(memoryLogger.getMemoryLogs()));
+  await ipcRenderer.invoke(
+    "save-logs",
+    path,
+    JSON.stringify(memoryLogger.getMemoryLogs(), null, 4),
```

Comment:
I like 2 spaces instead of 4, wdyt ?
```suggestion
    JSON.stringify(memoryLogger.getMemoryLogs(), null, 2),
```